lib/package.json

```diff
@@ -33,7 +33,7 @@
   },
   "peerDependencies": {
     "classnames": "^2.2.5",
-    "material-ui": "^1.0.0-beta.31",
+    "material-ui": "^1.0.0-beta.33",
```

Comment:
why 33?

Reply:
isn't it compatible with 32 too?

Reply:
Oh ( 
Yes but I would not publish additional release :)

Reply:
it would be great to have a possibility to review release PRs *before* release

Reply:
Sorry :) 
Just think that it would be just fast release with the fixes) 
Will assign pull request and wait review. My fault.

Reply:
no worries :)